### PR TITLE
MetaAnalytics: Force dashboard usage events emitter when the FF alwaysSendUsageInisghts is enabled

### PR DESCRIPTION
**Pull request by ivanortegaalba**
Add a new FF to enable dashboard usage analytics to emit usage events ignoring `config.analytics.enabled` value

### CODE REVIEW

File: pkg/services/featuremgmt/registry.go [hunk 1]
```diff
@@ -1362,6 +1362,12 @@ var (
 			Owner:           grafanaAlertingSquad,
 			RequiresRestart: true,
 		},
+		{
+			Name:        "alwaysSendUsageInisghts",
+			Description: "Forces sending usage insights data no matter what if config.analytics.enabled is enabled",
+			Stage:       FeatureStageExperimental,
+			Owner:       grafanaSharingSquad,
```
Comment (yaelleC) [inline pkg/services/featuremgmt/registry.go:1369]: should that be Enterprise instead?

File: pkg/services/featuremgmt/registry.go [hunk 2]
```diff
@@ -1362,6 +1362,12 @@ var (
 			Owner:           grafanaAlertingSquad,
 			RequiresRestart: true,
 		},
+		{
+			Name:        "alwaysSendUsageInisghts",
+			Description: "Forces sending usage insights data no matter what if config.analytics.enabled is enabled",
+			Stage:       FeatureStageExperimental,
```
Comment (yaelleC) [inline pkg/services/featuremgmt/registry.go:1368]:
```suggestion
			Stage:       FeatureStageExperimental,
			FrontendOnly:      true,
			AllowSelfServe:    false,
			HideFromDocs:      true,
			HideFromAdminPage: true,
```

File: pkg/services/featuremgmt/registry.go (hunk 1 above)
Comment (ivanortegaalba) [reply pkg/services/featuremgmt/registry.go:1369]: They don't have team in GH, and they don't appears in owners file

**Comment by danielkenlee** [comment]
Closing for now since we don't need to push a release for Inditex. @grafana/grafana-operator-experience-squad will take it from here.

**Pull request closed without merging**